modules/standard/FileSystem.chpl

```diff
@@ -265,8 +263,8 @@ proc chown(out error: syserr, name: string, uid: int, gid: int) {
 // When basename and joinPath are supported, enable dest to be a directory.
 
 /* Copies the contents and permissions of the file indicated by `src` into
-   the file or directory `dest`.  If `dest` is a directory, will throw a
-   FileNotFoundError.  If `metadata` is set to `true`, will also copy the
+   the file or directory `dest`.
+   If `metadata` is set to `true`, will also copy the
```

Comment:
The line wrapping after removing that information is weird, should be fixable with a quick Meta-Q in emacs